kore/test/Test/Kore/IndexedModule/Resolvers.hs

```diff
@@ -285,6 +295,11 @@ test_resolvers =
                 (getSentenceSymbolOrAliasHead objectB [])
             )
         )
+    , testCase "sort indexed axioms"
+        (assertEqual ""
+            (reverse $ List.sort [axiomA, axiomA'])
+            (getIndexedSentence <$> indexedModuleAxioms testIndexedObjectModule)
+        )
```

Comment:
I'd prefer something like:

```haskell
axiomsSortedBackward = [axiomSortedLast, axiomSortedFirst]
-- axiomSortedBackward is use to insert into `testObjectModel`

assertEqual
   assertEqual (reverse axiomSortedBackward)
   ....
```

I would also typically, if it's easy, have a test that shows that the naive way of extracting the elements would result in them being in the opposite order. Otherwise, there's a too high chance of the particular test being only coincidentally correct.

This also works nicely with TDD. A typical sequence would be:

1. Write a test that confirms that the function returns the elements in the order inserted (if it doesn't exist).
2. Preserve that test.
3. Write a new test that inserts the elements in the opposite order but it expects the same result as the earlier test.
4. That fails, forcing you to add sorting code. 
5. The two tests guard against coincidental correctness.